### ISSUE TITLE
Rasa X autogenerated lookup table is not written to disk

### ISSUE THREAD

**Issue by Krogsager** [RasaHQ/rasa#4859]
**Rasa version**: 1.4.5

**Rasa X version**:  0.22.2

**Python version**: 3.7.5

**Operating system**: Ubuntu 18.04.3 LTS

**Issue**: 
TL:DR lookup table files appear to be created, but are no actually written to disk. Kept in memory and disapperars after some time.

(**1**) When uploading an nlu.md file (which contains a lookuptable) in the gui ,  (**2**) a message informs me that I should upload this table separately.
(**3**)The lookup table then appears in the GUI and is available to download.

When I check the nlu file it has been modified:
```
## lookup:locations
  data/locations
```
 But `locations` does not exist. The only files in `/data/` are `domain.md` and `nlu.md`.
Only after downloading and (**4**) uploading the `locations` file is it written to the `/data/` dir.

![image](https://user-images.githubusercontent.com/27994384/69728211-5d4c5580-1124-11ea-9f46-4d49be063713.png)

As a solution I download `locations` and reupload it (**3**) & (**4**).

**How to recreated**: 
Upload an nlu file with a lookup in it. Then take a look in the `/data/` dir and see that no new lookup file is created.

**Additional bug**
I will make a new issue if I'm asked to.
I just noticed that Rasa X deletes my lookup table file if I upload an nlu file with a lookup file reference.
This nlu data:
```
## lookup:locations
  data/locations
```

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @tmbo will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by zoka123** [comment]
Also, when committing changes back to the Git server, RasaX removes previous references to these lookup tables:

![image](https://user-images.githubusercontent.com/3052111/72472345-ad6e9c80-37e4-11ea-97fe-270efb7f1eeb.png)

What's the right way to handle bigger lookup datasets if we want to work with Rasa X for bots in production?

**Comment by koszta5** [comment]
Even when uploading lookup tables separately they are dont have a reference added to nlu.md. This means that they are essentially completely ignored :( :( 
Please please fix this or provide workaround
